modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx

```diff
@@ -530,7 +561,12 @@ const ClaimsTable = () => {
 	};
 
 	const setSortRule = () => {
-		sortByDate === 'desc' ? setSortByDate('asc') : setSortByDate('desc');
+		sortedOrder === 'desc' ? setSortedOrder('asc') : setSortedOrder('desc');
```

Comment:
Could you use 'desc' and 'asc' as an `enum`.

Reply:
Yes, sure.
Done!